README.md

```diff
@@ -1 +1,42 @@
-# spitoruler
\ No newline at end of file
+# Spito
```

Comment:
```suggestion
# SPITO
```

Reply:
Instead of looking for expansion of the acronym, we can simply make it proper name.

Reply:
We should discuss it during our next meeting.